pom.xml

```diff
@@ -247,6 +247,11 @@
                 <artifactId>netty-reactive-streams-http</artifactId>
                 <version>2.0.0</version>
             </dependency>
+            <dependency>
+                <groupId>io.reactivex.rxjava2</groupId>
```

Comment:
What's the size of this dependency?

Reply:
Size is 2.1MB. I added it to show that customer can use 3rd party implementations for additional helper methods. I can remove it if we decide so.

Reply:
Is this a test dependency?

Reply:
Yes, I have marked the scope as test in services pom.

Reply:
No just where it's used is fine. Just making sure it's a test dependency.